vectorizers/_vectorizers.py

```diff
@@ -401,7 +401,7 @@ def preprocess_tree_sequences(
 
     # We will prune the edges from any nodes who's labels are to be filtered and reconnect their parents with their children.
     # This will remove them from our computation without having to alter the matrix size or label_sequence.
-    if not masking:
+    if masking == None:
```

Comment:
```suggestion
    if masking is None:
```
Using is for None is standard practice since None is a singleton object to checking for identity makes sense over equality.